### DISCUSSION TITLE
Content Design - resources

### DISCUSSION THREAD

**Discussion by kimbell27** [nhsbsa/nhsbsa-digital-playbook#299] [UCD]
[Dee made a good suggestion](https://github.com/nhsbsa/nhsbsa-digital-playbook/discussions/250) about publishing or linking to our style guide from the playbook. 

I wonder what other resources we might want to see in here? Would this be a general resources section or a section designed for new starters? Of if we have both, how can we avoid duplication? Just as a starter for 10 😉

**Comment by nhsbsa-sthil** [comment]
@kimbell27 @Dee-Powell

Morning folks! Just spotted this pop up and thought I'd mention that the NHS BSA Content Style Guide is linked within the playbook currently here:

https://nhsbsa.github.io/nhsbsa-digital-playbook/design/content-designer/about-role/#sources

Whether that's the right place for it though is obviously up for debate! 😄
